AUTHORS

```diff
@@ -2,4 +2,5 @@ Barnaby Court (bcourt@redhat.com)
 Michael Hrivnak (mhrivnak@redhat.com)
 Sayli Karmarkar (skarmark@redhat.com)
 Dennis Kliban (dkliban@redhat.com)
-Aaron Weitekamp (aweiteka@redhat.com)
+Aaron Weitekamp (aweiteka@redhat.com
```

Comment:
Sorry to be a perfectionist, but could you add the trailing ``)`` ? Thanks! Then I promise I'll merge it.